docs/install/docker.md

```diff
@@ -67,6 +67,9 @@ services:
     restart: always
 ```
 
+!!! note
+    If you are running [podman](https://podman.io/), please check the appropriate [page](./podman.md)
```

Comment:
```suggestion
    If you are running [Podman](https://podman.io/), please check the appropriate [page](./podman.md)
```

Reply:
Podman is with lowercase p as this is a mix between pod and manager